spec/models/bathroom_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+User.create!(
+  email: "email@email.com",
+  encrypted_password: "password",
+  password: "password"
+  )
+Bathroom.new(
+  user_id: 1,
+  rating_average: 3,
+  location_name: "Starbucks",
+  street_address: "62 Boylston",
+  zip_code: 02116,
+  description: "This is the best bathroom ever!"
+  )
```

Comment:
I don't think this code is necessary. I would remove the `User.create!` and `Bathroom.new` calls.